internal/endpoints/endpoints.go

```diff
@@ -38,6 +39,19 @@ func (e *Endpoints) Up() error {
 	return nil
 }
 
+// UpdateTLS updates the TLS configuration of the network listeners.
+func (e *Endpoints) UpdateTLS(cert *shared.CertInfo) {
+	e.mu.Lock()
+	defer e.mu.Unlock()
+
+	for _, l := range e.listeners {
+		n, ok := l.(*Network)
+		if ok {
+			n.UpdateTLS(cert)
+		}
```

Comment:
Is there any way to log an error if the casting didn't work? Not sure if applicable :)

Reply:
So not every `Endpoint` is a `Network` type, because there's also unix socket listeners which we want to quietly skip in this loop. Additionally, if the daemon isn't initialized yet, there may be no Network listener at all yet. It would still be valid to change the cluster cert at this point, so we shouldn't necessarily error out if we can't find the Endpoint.

Reply:
Ok, thanks!